core/sql/optimizer/opt.cpp

```diff
@@ -5034,6 +5029,20 @@ void OptDefaults::initialize(RelExpr* rootExpr)
 	  preFetchHistograms_           = FALSE;
   }
 
+  // Find out what the RMS security key invalidation garbage collection
+  // interval is (this mirrors the logic in runtimestats/ssmpipc.cpp), so
+  // we can fail-safe the histogram cache.
+
+  char *sct = getenv("RMS_SIK_GC_INTERVAL_SECONDS");  // in seconds
+  if (sct)
+    {
+      siKeyGCinterval_ = ((Int64) str_atoi(sct, str_len(sct)));
+      if (siKeyGCinterval_ < 10)
+        siKeyGCinterval_ = 10;
+    }
+  else
+    siKeyGCinterval_ = (Int64)24 * 60 * 60; // 24 hours
+
```

Comment:
Environment variable values are inherited at process create time. I suppose it is possible that we could write code in our processes to change it but that seems a devilish thing to do. I've followed the same design used for caching CQD values in opt.cpp.